### PR TITLE
Update isTimeOut to support edge case (when the update() dosen't get called)

### PR DESCRIPTION
**Pull request by twxia**

### CODE REVIEW

**Review by tailingchen — APPROVED** [review]
lgtm

File: contracts/ChainlinkPriceFeedV3.sol [hunk 1]
```diff
@@ -82,7 +82,13 @@ contract ChainlinkPriceFeedV3 is IChainlinkPriceFeedV3, IPriceFeedUpdate, BlockC
     }
 
     function isTimedOut() external view override returns (bool) {
-        return _lastValidTimestamp > 0 && _lastValidTimestamp.add(_timeout) < _blockTimestamp();
+        (, uint256 lastestValidTimestamp) = _getCachePrice();
```
Comment (tailingchen) [inline contracts/ChainlinkPriceFeedV3.sol:85]: add comment
// To fetch the latest price to prevent the cached _lastValidTimestamp is outdated.

Comment (twxia) [reply contracts/ChainlinkPriceFeedV3.sol:77]: just found that this logic is still incorrect. fixed in 87802cb

**Review by tailingchen — APPROVED** [review]
lgtm

**Pull request merged**